doc/gh-pages/pages/download/download.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Download
+has_children: true
+nav_order: 2
+---
+# Download MNE-CPP
+
+<html>
+<div style="height:13px"></div>
+</html>
+
+Stable Releases
+{: .label .label-blue }
+
+| Version | Release | Download link |
+|-------|-------|-------|
+| [0.1.8](changelog.md#version-018) | 2020-11-19 | <span class="fs-2"> [Windows](https://github.com/mne-tools/mne-cpp/releases/download/v0.1.8/mne-cpp-windows-dynamic-x86_64.zip){: .btn .btn-blue } [Linux](https://github.com/mne-tools/mne-cpp/releases/download/v0.1.8/mne-cpp-linux-dynamic-x86_64.tar.gz){: .btn .btn-blue } [MacOS](https://github.com/mne-tools/mne-cpp/releases/download/v0.1.8/mne-cpp-macos-dynamic-x86_64.tar.gz){: .btn .btn-blue } </span> |
+
+See the [changelog](changelog.md) to see what features where added in each release version.
+
+---
+
+This is the quickest way to get MNE-CPP up and running on your machine.
+
+### 1. Download MNE-CPP.
+
+Select your version and platform. Download the latest stable release for the best stability.
+
+### 2. Unzip the downloaded file.
+Extract the downloaded build into a folder on your machine
+
+### 3. Select the tool you wish to use from the `bin` folder.
+
+In the `bin` folder,  you will find:
+  - A real-time analysis application: [MNE-Scan](../documentation/scan.md).
+  - An offline visualization/analysis application: [MNE Analyze](../documentation/analyze.md).
+  - An anonymization/deidentification tool for FIF files: [MNE Anonymize](../documentation/anonymize.md).
+
+---
+
+Using the latest stable release is recommended for users who wish to use MNE-CPP's command line tools and GUIs. But if you want to checkout the latest features, try downloading the [development versions](../development/development.md).
```

Comment:
'Checkout' should be 'check out', unless this is a git pun, in which case i'm all for it.

Reply:
Thank you very much.